### PR TITLE
Past speakers

### PR DESCRIPTION
**Pull request by Folashade**
Changes
=======
- Created a past-speakers page (but did not add it to nav)
- Added a link to past speakers under the home:schedule and under the speak page FAQ
- on the past speakers page, there is a link to submit a talk
- added styles 😋

### CODE REVIEW

**Comment by matthewmcvickar** [comment]
@Folashade Wow, thank you so much for adding this! Happy Birthday! 😄🎉

**Comment by matthewmcvickar** [comment]
@Folashade Wow, thank you so much for adding this! Happy Birthday! 😄🎉

File: css/main.css [hunk 1]
```diff
@@ -423,6 +424,23 @@ button {
...
+}
+
+.past-event {
+    list-style-type: none;
+    position: relative;
+    padding-left: 20px;
+}
+
+.past-event .event-speaker p {
+    color: #779;;
```
Comment (paulcpederson) [inline css/main.css]: got an extra semi hangin out here, FYI

Comment (matthewmcvickar) [inline css/main.css]: Thanks for the sharp eyes, Paul! I'll remove that and add the rest of the past speakers soon. :)

Comment (paulcpederson) [inline css/main.css]: :+1:

Comment (Folashade) [inline css/main.css]: that sneaky semi 😈

**Pull request merged**